rds_disaster_recovery/rds_aurora_cluster_restore.py

```diff
@@ -0,0 +1,83 @@
+import time
+from datetime import datetime, timedelta
+
+import boto3
+import operator
+import sys
+
+client_us_west_2 = boto3.client("rds", region_name="us-west-2")
```

Comment:
similar thoughts, maybe again ask via the script to provide the region and then perform the operations

Reply:
Its Done.